backend/docker-compose.yml

```diff
@@ -10,8 +10,7 @@ services:
         env_file:
             - ./.env
         ports:
-            - 127.0.0.1:5986:5986
-            - 127.0.0.1:5984:5984
+            - 127.0.0.1:${COUCHDB_PORT}:${COUCHDB_PORT}
```

Comment:
no, no, no
the last port is static, cannot be changed

Reply:
127.0.0.1:${COUCHDB_PORT}:5986

Reply:
done